Trappist/src/Promact.Trappist.Web/wwwroot/app/profile/profile-dashboard/change-password-dialog.html

```diff
@@ -32,7 +36,7 @@ <h1 class="pull-left" md-dialog-title>Change Password</h1>
         </md-dialog-content>
         <md-dialog-actions>
             <button type="button" class="btn" md-dialog-close>Cancel</button>
-            <button type="submit" class="btn btn-primary" [disabled]="!passwordForm.form.valid" (click)="changePassword(user)">Save</button>
+            <button type="submit" class="btn btn-primary" [disabled]="!passwordForm.form.valid">Save</button>
```

Comment:
Ma'm I had added form tag in this html.so when submit button is clicked ngSubmit will automatically cal the changePassword() function ..that is y I removed it from here.
<form (ngSubmit)="changePassword(user)" #passwordForm="ngForm">
This line I am calling the same function.